lua/packer/plugin_types/git.lua

```diff
@@ -454,6 +491,30 @@ git.setup = function(plugin)
     end)()
     return r
   end
+
+  ---Reset the plugin to `plugin.commit`
+  plugin.revert = function ()
+    local log = require('packer.log')
+    if plugin.commit ~= nil then
+      return async(function ()
+        log.info(fmt("Reverting '%s' to commit '%s'", plugin.name, plugin.commit))
+        await(reset(install_to, plugin.commit))
+      end)
+    end
+
+    return async(function ()
+        log.info(fmt("'%s' has no commit to revert to", plugin.name))
+    end)
+  end
+
+  ---Returns HEAD's short hash
+  ---@return string
+  plugin.get_rev = function()
+    return async(function()
+      local res = await(get_rev(plugin))
```

Comment:
Nitpick: this wrapper is unnecessary - it's async and needs to be awaited anyway, so we can just have its callers await the `get_rev` function (or move that logic here).